lib/rmt/import_script.rb

```diff
@@ -0,0 +1,161 @@
+require 'csv'
+require 'ostruct'
+
+
+class ImportScript
```

Comment:
It's not a script, it's a class now. Rename to something like `SMTImporter`, perhaps?